File: test/www/domain2/cgi/result.py
#!/usr/bin/python3

# Import modules for CGI handling
import cgi, cgitb
import time

# Create instance of FieldStorage
form = cgi.FieldStorage()

# Get data from fields
postContent = form.getvalue('contentPost')
getContent = form.getvalue('contentGet')

print("Set-Cookie:lastvisit=" + str(time.time()))
print("Content-type:text/html\r\n\r\n")
print("<html>")
print("<head>")
print("<title>Hello - CGI Program</title>")
print("</head>")
print("<body>")
print("<h2>post %s</h2>" % (postContent))
print("<h2>get %s</h2>" % (getContent))
print("</body>")
print("</html>")

File: test/www/domain2/cgi/result.php
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>HTML Result Page for Testing</title>
</head>

<body>

<h1>Testing result</h1>

<p>content post : <?php echo $_POST["contentPost"] ?></p>
<p>content get : <?php echo $_GET["contentGet"] ?></p>

<?php
    echo '<pre>';
    var_dump($_ENV);
    echo '</pre>';

    echo '<pre>';
    var_dump($_POST);
    echo '</pre>';

    echo '<pre>';
    var_dump($_GET);
    echo '</pre>';
?>

</body>

</html>
